### DISCUSSION TITLE
Feature request: keyboard shortcut for 'Ignore Overrides'

### DISCUSSION THREAD

**Discussion by CamDawg** [NearInfinityBrowser/NearInfinity#172] [Ideas & Feature Requests]
I like the new Preferences menu, but one thing I miss is the easy access to toggle 'Ignore Overrides'. Previously it was two clicks (Tools > Ignore Overrides); now it's five (Game > Preferences > Ignore Overrides > Apply > Confirm). It's a setting I use on the EEs especially to check changes during mod development and it would be nice to have a shortcut for it, assuming it can be done without breaking the new menu.

**Comment by CamDawg** [comment]
Now that I think about it, a better option might be a context menu option. Having 'Open original' under 'Open in new window' in the asset tree context menu would also work very well. Or is 'backup', not 'original', the preferred NI nomenclature?

**Reply by Argent77** [reply]
I'll see what I can do, but it's probably a time-consuming process since this feature would require some fundamental design changes to the handling of game resources. Currently high-level resource access is controlled by a global flag that has to be switched on or off.

**Reply by CamDawg** [reply]
No worries then, I can cope. You never know if a request is "sure, that's two lines of code" or "that would require a team of coders eight weeks".

**Reply by Argent77** [reply]
I have implemented this feature as a new menu entry in File menu and context menu for the resource tree. I'd consider this feature 'experimental' as the implementation might have introduced side effects I haven't noticed yet.

It can be tested in the [Nightly Release](https://github.com/Argent77/NearInfinity/releases/tag/nightly).
